CAT/model/IRT.py

```diff
@@ -277,6 +284,86 @@ def get_fisher(self, student_id, question_id, pred_all):
         q = 1 - pred
         fisher_info = (q*pred*(alpha * alpha.T)).numpy()
         return fisher_info
+    def IRT_derivate(self,pred_all):
```

Comment:
Add an empty line before this. Please also make sure the format is correct of all your code